### ISSUE TITLE
Debug check crashes on iOS 9 Beta

### ISSUE THREAD

**Issue by editdb** [project-imas/security-check#8]
If you peruse https://nabla-c0d3.github.io/blog/2015/06/16/ios9-security-privacy/ you’ll see the following section:
- - - - - - 
Apple has closed three privacy gaps that allowed Apps to detect which other Apps were installed on the device.
The first technique was to use the sysctl() function to retrieve the process table (a remnant of OS X), which includes the list of running Apps. In iOS 9, sysctl() was modified to no longer allow sandboxed Apps to retrieve information about other running processes.
- - - - - - 
IMAS uses sysctl to detect the pid. 

Solution:
The IMAS function dbgGetPid() can be replaced with the built-in function getpid()

**Comment by rolmaz** [comment]
i have the same issue

**Comment by ghe0106** [comment]
Yup - me too - can u provide a code example of the above solution ?

**Comment by editdb** [comment]
Search for dbgGetPid() and replace it with getpid()

You'll find it within SecurityCheck.xcodeproj > debugCheck.h module debugCheck(cb)

Be careful not to remove the backslash from the end of the line

**Comment by Emmo213** [comment]
Since I don't trust Apple I added an if check in my code where it'll use getpid on iOS9 but keep dbgGetPid on older versions.

**Comment by ghe0106** [comment]
Got that to work thanks !

**Comment by mesutoktem** [comment]
This solution does not work for me.

**Comment by megavolt605** [comment]
@mmesutt thank you for your patch.
